ecs-deploy

```diff
@@ -425,10 +425,10 @@ function updateServiceForceNewDeployment() {
 function updateService() {
     if [[ $(echo ${NEW_DEF} | jq ".containerDefinitions[0].healthCheck != null") == true ]]; then
         checkFieldName="healthStatus"
-        checkFieldValue="HEALTHY"
+        checkFieldValue='"HEALTHY"'
     else
         checkFieldName="lastStatus"
-        checkFieldValue="RUNNING"
+        checkFieldValue='"RUNNING"'
```

Comment:
Does that work? It looks like it will do a grep =n ""HEALTHY""  (with two sets of double quotation marks)

Reply:
I see you already tested it.